app/controllers/select_documents_variant_c_controller.rb

```diff
@@ -53,4 +53,15 @@ def report_user_evidence_to_piwik(selected_evidence)
   def increase_attempt_number
     session[:evidence_attempt_number] = (session[:evidence_attempt_number] || 0) + 1
   end
+
+  # HUB-702 The content designers asked us to combine credit 
+  # card and driving license for the bullet points under 
+  # specific circumstances.  So thats what this methods does
```

Comment:
```suggestion
  # specific circumstances.
```

Non-urgent - don't allow this tiny amendment to delay the merge.